Makefile

```diff
@@ -23,10 +23,13 @@ clean:
 	find . -name '__pycache__' -exec rm -rf {} +
 
 test:
-	tox
+	export COMPOSE_FILE=docker-compose-dev.yml; docker-compose run --rm flaskbb tox
```

Comment:
Please create a new Makefile entry instead of changing this one to run with docker

Reply:
Okay i want a do this changes and update the PR